clang/unittests/Format/FormatTest.cpp

```diff
@@ -26588,10 +26613,7 @@ TEST_F(FormatTest, Cpp20ModulesSupport) {
                "  int foo;\n"
                "};",
                Style);
-  verifyFormat("export {\n"
-               "  int foo;\n"
-               "};",
-               Style);
+  verifyFormat("export { int foo; };", Style);
```

Comment:
You can't fix the tests by changing them... here you make the decision that people want a collapsed export {} what makes you say thats what everyone wants?

Reply:
Yeah, makes sense. As far as I understand it, were were previously just parsing this as a compound statement, which afaik isn’t formatted on a single line by default, but maybe namespaces are, but I’m candidly not quite sure what’s causing this to be formatted on one line... I think this has something to do with the fact that I used `parseBlock` for this, but I’ll have to look into it a bit more.

Do we want a separate option for this (e.g. something like `AllowShortExportBlocksOnASingleLine`) or should that just fall under `AllowShortBlocksOnASingleLine`?

Reply:
There should be an option to en/disable it, if it can be shared with other things, or be exclusively for export is to be discussed. I have no strong opinion, but would lean to an extra option, because someone will ask for it.

Reply:
Makes sense; I’ll hopefully have time to look into this next week because I’ve been a bit busy recently.